README.md

```diff
@@ -24,7 +24,5 @@ To use this app, follow these steps:
 1. Clone and run Balistos API.
 2. Install webpack globally with node command '**npm install webpack webpack-dev-server -g**' .
 3. Run node command '**npm install**' to install all necessary node.js modules.
-4. Run app with '**npm run start**'. App will by default start listening to port 8080.
-
-
-
+4. Change API_INDEX to '***http://localhost:4000***' in the index.jsx file.
+5. Run app with '**npm run start**'. App will by default start listening to port 3000.
```

Comment:
Yes, I know. With current configuration, API is on port 3000 and when you start React app, it says port 3000 is already in use and it has to choose a random port to run React part on (not 8080 as noted in instructions).